### DISCUSSION TITLE
Is it possible to customize the suffix of modifier from ':' to '_'

### DISCUSSION THREAD

**Discussion by zhengpq** [tailwindlabs/tailwindcss#7450] [Help]
just like the question above，in my project，having ’:‘ in class is not allowed，but modifiers in tailwind use ':' in class, so i want to know how to change the ':' to others i want, just like blow:

in tailwind now:
```
first:mp-0
``` 

what i want:
```
first_mp-0
```

**Comment by zhengpq** [comment]
in tailwind now:

```
<div class="first:mp-0"></div>
```

what i want:
```
<div class="first_mp-0"></div>
```

**Comment by adamwathan** [comment]
Yep, here: https://tailwindcss.com/docs/configuration#separator

**Reply by zhengpq** [reply]
Thank you so much!
